addons/mail/static/src/messaging/component/partner_im_status_icon/partner_im_status_icon.js

```diff
@@ -36,6 +36,10 @@ class PartnerImStatusIcon extends Component {
 Object.assign(PartnerImStatusIcon, {
     props: {
         partnerLocalId: String,
+        partnerMentionSuggestionDisplay: {
+            type: Boolean,
+            optional: true,
```

Comment:
Boolean should usually not be optional, because it would imply the code differentiate `false` and `undefined`. Instead we can add it in `defaultProps` with value `false`.